streams/streams-scala/src/test/scala/org/apache/kafka/streams/scala/KStreamTest.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2018 Joan Goyeau.
```

Comment:
Is it good to put my name here?

Reply:
I'm pretty sure this is binary compatible.